.github/workflows/stale-bot.yml

```diff
@@ -0,0 +1,15 @@
+name: "Close stale PRs"
+on:
+  schedule:
+    - cron: "30 1 * * *"
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v9
+        with:
```

Comment:
do we want to check also stale issues?

Reply:
I'm not sure about that so I asked the question here https://github.com/gnolang/gno/issues/1445#issuecomment-2352294634